client/me/account/main.jsx

```diff
@@ -656,10 +658,15 @@ class Account extends Component {
 					userSettings={ this.props.userSettings }
 				/>
 
-				<FormFieldset>
-					<FormLabel htmlFor="primary_site_ID">{ translate( 'Primary site' ) }</FormLabel>
-					{ this.renderPrimarySite() }
-				</FormFieldset>
+				{ this.props.defaultInterface === DefaultInterface.PRIMARY_SITE && (
+					<FormFieldset>
+						<FormLabel htmlFor="primary_site_ID">{ translate( 'Primary site' ) }</FormLabel>
+						<FormSettingExplanation>
+							{ translate( "Select the site whose dashboard you'll see first when logging in." ) }
```

Comment:
```suggestion
							{ translate( "Choose the default site dashboard you'll see at login." ) }
```

Alternative:

- Set your primary site dashboard for quicker access.